_includes/common/experimental-warning.md

```diff
@@ -0,0 +1,3 @@
+{{site.data.alerts.callout_danger}}
```

Comment:
We already have the concept of "experimental feature" and have an include that we use in some places, although it's versioned and shouldn't be: https://github.com/cockroachdb/docs/blob/master/_includes/v21.2/misc/experimental-warning.md. We also have a page that lists experimental features, though it's not exposed in the sidenav for some reason: https://www.cockroachlabs.com/docs/v21.2/experimental-features.html.

All that said, if experimental is the right categorization (and it does sound right, based on the similarities between your include and the old one), how do we combine experimental with "restricted", which is a cloud-only phenomenon, I think? cc @marksoper

Reply:
Yes, good point @jseldess - in this case it is experimental but has the additional property of being restricted.

Reply:
So it sounds like we should have a separate, Cloud-specific warning for private/restricted experimental features, as there aren't any cases where CRDB would have a private feature.

Reply:
+1 on having a separate message for this - I think we need to differentiate the two. 

The other "experimental features" message sounds more like the actual A/B experiments that run and our users could be in the experimentation group to get them. 

For the API feature, I like the terminology of private/restricted as these are opt-in and selected orgs that we chose for them.

Reply:
Ok, I created a separate Cloud-specific warning.